components/tabs/Tabs.module.scss

```diff
@@ -37,13 +53,18 @@
 
 	&.active {
 		color: $primary-color4 !important;
+
+		> [data-elem='base-cell'] {
+			> [data-elem='component1'] {
+				> .icon {
+					fill: $highlight;
```

Comment:
not `$highlight`. use correct hexcode.

Reply:
This is correct, verified from the design